tests/e2e/env/setup.sh

```diff
@@ -20,12 +20,12 @@ if [[ $FORCE_E2E_DEPS_SETUP || ! -d $SERVER_PATH ]]; then
 	fi
 
 	rm -rf $SERVER_PATH
```

Comment:
@leonardola Should we add quotes here, [here](https://github.com/Automattic/woocommerce-payments/blob/3e499b0499b2e14306a16c7654df61adf4419c6c/tests/e2e/env/setup.sh#L14) and for the `$DEV_TOOLS_PATH` as well?